src/Microsoft.Data.Analysis/DataFrameColumn.cs

```diff
@@ -185,9 +185,20 @@ public virtual DataFrameColumn Sort(bool ascending = true)
         public virtual bool HasDescription() => false;
 
         /// <summary>
-        /// Returns a DataFrame with statistics that describe the column
+        /// Returns a <seealso cref="StringDataFrameColumn"/> containing the DataType and Length of this column
         /// </summary>
-        public virtual DataFrame Description() => throw new NotImplementedException();
+        public virtual StringDataFrameColumn Info()
+        {
+            StringDataFrameColumn dataColumn = new StringDataFrameColumn(Name, 2);
+            dataColumn[0] = DataType.ToString();
+            dataColumn[1] = (Length - NullCount).ToString();
+            return dataColumn;
+        }
+
+        /// <summary>
+        /// Returns a <see cref="PrimitiveDataFrameColumn{T}"/> with statistics that describe the column
+        /// </summary>
+        public virtual PrimitiveDataFrameColumn<float> Description() => throw new NotImplementedException();
```

Comment:
I wonder if the signature shouldn't be `DataFrameColumn` instead of a concrete type.